openapi/dcos.yaml

```diff
@@ -2163,9 +2163,10 @@ components:
       properties:
         command:
           $ref: '#/components/schemas/command'
-        config:
-          additionalProperties: true
-          type: object
+        # Causes a JSON unmarshal error at the moment, config should be defined as a JSON schema instead.
+        # config:
```

Comment:
In all our uses so far, this is a critical property of the response: this is the only place it is available, not via package search response, etc. I wouldn't want to see this merged without this being available via this call.

Reply:
This works if you remove the `additionalProperties: true` from `config` declaration (generated code is `map[string]interface{}` which works great for now.

Reply:
It worked out indeed :+1: